cmd/atlas/internal/cmdapi/migrate.go

```diff
@@ -42,6 +42,7 @@ const (
 	migrateFlagFormat          = "format"
 	migrateFlagLog             = "log"
 	migrateFlagRevisionsSchema = "revisions-schema"
+	migrateFlagCreateRevisions = "create-revisions"
```

Comment:
Maybe, maybe, `rev-schema` and `create-rev-table` work better together.

Reply:
Create rev table

Reply:
`create-revisions-table`? I don't like abbreviations in parts of a flag.